src/Routes/GSheetImport.ts

```diff
@@ -4,6 +4,8 @@ import { gSheetDataFetcher } from "../utils/gSheetDataFetcher"
 import { sanitizeRow } from "../utils/sanitizeRow"
 import { updateDatabase } from "../utils/updateDatabase"
 
+const { SPREADSHEET_IDS_WHITELIST } = process.env
```

Comment:
Slight preference for "allowlist" language as discussed recently in #dev.

Reply:
Yup, agreed. That was a slip on my part. Will update